docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,43 @@
+## Authentication
+Plio Analytics uses Auth0 authentication to manage access tokens and valid API requests from the client app, which in Plio's ecosystem is the frontend app.
```

Comment:
```suggestion
Plio Analytics uses Auth0 authentication to manage access tokens and valid API requests from a client app.
```
I don't think it's needed. Any client app can connect to this analytics service right?
Just an opinion though!